g_warning makes gtester abort tests. We do not care about this
particular one because it actually is in ${T} which is a safe directory
with regard to live system.

--- a/tests/ephy-file-helpers-test.c	2012-09-10 17:39:58.000000000 +0200
+++ b/tests/ephy-file-helpers-test.c	2013-01-20 22:22:27.409706652 +0100
@@ -96,8 +96,6 @@
        * prefixed with /tmp. */
       if (g_str_has_prefix (tmp_dir, "/tmp"))
         g_assert (ephy_file_delete_dir_recursively (file, NULL));
-      else
-        g_warning ("INIT: dangerous path returned as tmp_dir: %s", tmp_dir);
 
       g_object_unref (file);
     }
